BrainPortal/app/models/cluster_task.rb

```diff
@@ -1470,7 +1470,23 @@ def submit_cluster_job
 
 # CbrainTask '#{self.name}' commands section
 
-#{commands.join("\n")}
+#{
```

Comment:
This entire string interpolation is horrible, Tristan!

Please rewrite cleanly, or let me do it later this week.